app/javascript/article-form/articleForm.jsx

```diff
@@ -43,6 +43,16 @@ export default class ArticleForm extends Component {
     }
   }
 
+  static handleEmgithubPreview() {
```

Comment:
refactor (blocking): The article form's purpose it to edit a post. It has no knowledge of liquid tags. Take a peek at existing liquid tags, like this one https://github.com/forem/forem/blob/d9011f4b593469d60499900660818e8ee897c382/app/liquid_tags/user_subscription_tag.rb